### DISCUSSION TITLE
这个深夜，Apple 特别活动来点亮。

### DISCUSSION THREAD

**Discussion by QuasarRyan** [share121/inter-knot#4694] [科技]
![](https://github.com/user-attachments/assets/17dc6a5e-b9bc-48f5-b928-0c90fae58789)
<p align="center">北京时间‌ 2024‌‌ 年 9‌‌ 月 10‌ 日凌‌晨 1‌ 点，Apple 特别活动</p>
<p align="center">在<a href="https://www.apple.com.cn/">apple.com</a> 直播，配有中文字幕。</p>
<h4 align="center">
<a href="https://www.apple.com.cn/apple-events/" title="event">进一步了解</a>
</h4>

**Comment by lllliufeng** [comment]
啊？

**Comment by Signedinn** [comment]
看完了 感觉那个新ai确实挺好的 还有以前一直想要的连接ChatGPT现在也有了 可惜欧盟还需要等几个月才能用
![IMG_4255](https://github.com/user-attachments/assets/e627b81b-e0e9-4ae8-8fb6-bd5d53e4d635)
